### PR TITLE
Rokeates patch 2

### PR DESCRIPTION
**Pull request by rokeates**
updates to monetization docs

### CODE REVIEW

File: docs/monetize-addins-through-microsoft-commercial-marketplace.md [hunk 1]
```diff
@@ -101,18 +100,19 @@ Your add-in should have three states:
...
 
-## Piloting your SaaS offer
+## Testing your SaaS offer
 
-You can submit your SaaS offer as a private plan - this will make your offer private and visible only to the restricted audience of your choosing. After you test and get feedback, you can update the audience or choose to make the plan available to everyone. When a plan is published as visible to everyone, it must remain visible to everyone - you can't configure it to be a private plan again.
+To test your integration, you should submit an offer to a private audience for $0 in order to ensure that the end to end process is working. A private plan makes your offer visible only to a restricted audience of your choosing.
+When you have verified that your solution is working, you should then submit the offer privately again, this time to prepare to go live. After you test and get feedback, you can update the audience or choose to make the plan available to everyone. When a plan is published as visible to everyone, it must remain visible to everyone - you can't configure it to be a private plan again.
```
Comment (Lauragra) [inline docs/monetize-addins-through-microsoft-commercial-marketplace.md]: Is "plan" the right word here, or are we referring to the SaaS offer itself?

File: docs/monetize-addins-through-microsoft-commercial-marketplace.md [hunk 2]
```diff
@@ -101,18 +100,19 @@ Your add-in should have three states:
...
 
 ## Submission process for SaaS offer + add-in
-You should submit your add-in first, with test accounts (admin + non-admin). After your add-in has been approved, you can then submit your SaaS offer. 
+Submit your SaaS offer to Partner Center as a private plan first. This ensures that customers don't purchase your SaaS app before the add-in is available. After your SaaS offer is approved, it will be assigned a GUID. Next, submit your add-in to Seller Dashboard and include this GUID in the test notes, as well as test accounts (admin + non-admin). After your add-in has been approved and is in the store, you can update your plan to be available to a public audience.
```
Comment (Lauragra) [inline docs/monetize-addins-through-microsoft-commercial-marketplace.md]: Okay to refer to Seller Dashboard here? We'll change this to Partner Center at GA (10/21)?

File: docs/monetize-addins-through-microsoft-commercial-marketplace.md (hunk 1 above)
Comment (rokeates) [reply docs/monetize-addins-through-microsoft-commercial-marketplace.md]: as far as I understand, the offer has a 'plan type'. I think plan is the right word but not necessarily the only word

File: docs/monetize-addins-through-microsoft-commercial-marketplace.md (hunk 2 above)
Comment (rokeates) [reply docs/monetize-addins-through-microsoft-commercial-marketplace.md]: Ah I'm not sure. If we made it PC now would it be confusing?

File: docs/monetize-addins-through-microsoft-commercial-marketplace.md (hunk 1 above)
Comment (Lauragra) [reply docs/monetize-addins-through-microsoft-commercial-marketplace.md]: Okay, sounds good.

**Pull request merged**